File: Java/TripTalk/src/main/java/com/mycompany/triptalk/clases/Respuesta.java

package com.mycompany.triptalk.clases;

public class Respuesta {
    private int idRespuesta;
    private int idPregunta;
    private String respuesta;
    private int idUsuario;

    public Respuesta() {
    }

    public Respuesta(int idRespuesta, int idPregunta, String respuesta, int idUsuario) {
        this.idRespuesta = idRespuesta;
        this.idPregunta = idPregunta;
        this.respuesta = respuesta;
        this.idUsuario = idUsuario;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public int getIdRespuesta() {
        return idRespuesta;
    }

    public void setIdRespuesta(int idRespuesta) {
        this.idRespuesta = idRespuesta;
    }

    public int getIdPregunta() {
        return idPregunta;
    }

    public void setIdPregunta(int idPregunta) {
        this.idPregunta = idPregunta;
    }

    public String getRespuesta() {
        return respuesta;
    }

    public void setRespuesta(String respuesta) {
        this.respuesta = respuesta;
    }
    
    
    
}

File: C#/TripTalk/TripTalk/Clases/Respuesta.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripTalk.Clases
{
    public class Respuesta
    {
        private int idRespuesta;
        private int idPregunta;
        private String respuesta;
        private int idUsuario;

        private Respuesta()
        {
        }
        public Respuesta(int idRespuesta, int idPregunta, string respuesta, int idUsuario)
        {
            this.idRespuesta = idRespuesta;
            this.idPregunta = idPregunta;
            this.respuesta = respuesta;
            this.idUsuario = idUsuario;
        }
        [JsonProperty("idRespuesta")]
        public int IdRespuesta { get => idRespuesta; set => idRespuesta = value; }
        [JsonProperty("idPregunta")]
        public int IdPregunta { get => idPregunta; set => idPregunta = value; }
        [JsonProperty("respuesta")]
        public string Answer { get => respuesta; set => respuesta = value; }
        [JsonProperty("idUsuario")]
        public int IdUsuario { get => idUsuario; set => idUsuario = value; }
    }
}
